### PR TITLE
Movies List was created

### PR DESCRIPTION
**Pull request by vladgusakovskiy**
[DEMO LINK](https://vladgusakovskiy.github.io/react_movies-list/)

### CODE REVIEW

File: src/components/MovieCard/MovieCard.tsx [hunk 1]
```diff
@@ -1,7 +1,52 @@
...
+            {movie.description}
+          </p>
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+          <a
+            href="https://www.imdb.com/title/tt1375666"
+            data-cy="MovieLink"
+          >
```
Comment (OleksiiPustovit) [inline src/components/MovieCard/MovieCard.tsx:45]: Maybe it's better to use here
```suggestion
          <a
            href={imdbUrl}
            data-cy="MovieLink"
          >
```

**Review by OleksiiPustovit — CHANGES_REQUESTED** [review]
Good start but I have one suggestion

**Review by OleksiiPustovit — APPROVED** [review]
👍

File: src/components/MovieCard/MovieCard.tsx [hunk 2]
```diff
@@ -1,7 +1,52 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type MovieType = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<MovieType> = ({ movie }) => {
```
Comment (denys-danyliuk) [inline src/components/MovieCard/MovieCard.tsx:9]: you can destructure movie 
```suggestion
export const MovieCard: React.FC<MovieType> = ({ movie }) => {
  const { 
    title,
    description, 
    imgUrl, 
    imdbUrl 
   } = movie;
```

**Review by denys-danyliuk — APPROVED** [review]
gj! pay attention to the comment above

**Pull request closed without merging**